src/main/java/dk/xam/jbang/Main.java

```diff
@@ -1,143 +1,21 @@
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.PrintWriter;
-import java.net.URISyntaxException;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.FileSystems;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.StandardWatchEventKinds;
-import java.nio.file.WatchEvent;
-import java.nio.file.WatchKey;
-import java.nio.file.WatchService;
-import java.security.KeyManagementException;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
-import java.security.cert.X509Certificate;
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.Stack;
-import java.util.concurrent.Callable;
-import java.util.jar.Attributes;
-import java.util.jar.JarFile;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
-
-import javax.net.ssl.HostnameVerifier;
-import javax.net.ssl.HttpsURLConnection;
-import javax.net.ssl.SSLContext;
-import javax.net.ssl.SSLSession;
-import javax.net.ssl.TrustManager;
-import javax.net.ssl.X509TrustManager;
-
-import org.apache.commons.text.StringEscapeUtils;
-import org.jboss.shrinkwrap.resolver.api.maven.Maven;
-import org.jboss.shrinkwrap.resolver.api.maven.coordinate.MavenCoordinate;
 
-import com.sun.nio.file.SensitivityWatchEventModifier;
+import dk.xam.jbang.cli.Jbang;
 
-import io.quarkus.qute.Template;
-import picocli.AutoComplete;
 import picocli.CommandLine;
-import picocli.CommandLine.Model.ArgSpec;
-import picocli.CommandLine.Model.CommandSpec;
-import picocli.CommandLine.Model.OptionSpec;
-
-abstract class JbangBaseCommand implements Callable<Integer> {
-
-	static {
-		Logger logger = Logger.getLogger("org.jboss.shrinkwrap.resolver");
-		logger.setLevel(Level.SEVERE);
-	}
-
-	@Spec
-	CommandSpec spec;
-
-	@Option(names = { "-h", "--help" }, usageHelp = true, description = "Display help/info")
-	boolean helpRequested;
-
-	static boolean verbose = false;
-
-	void info(String msg) {
-		spec.commandLine().getErr().println("[jbang] " + msg);
-	}
-
-	void warn(String msg) {
-		info("[WARNING] " + msg);
-	}
-
-	boolean isVerbose() {
-		return verbose;
-	}
-
-	@Override
-	public Integer call() throws IOException {
-		try {
-			return doCall();
-		} catch (ExitException e) {
-			if (isVerbose()) {
-				e.printStackTrace();
-			} else {
-				info(e.getMessage());
-				info("Run with --verbose for more details");
-			}
-
-			return e.getStatus();
-		}
-	}
-
-	public abstract Integer doCall() throws IOException;
-}
-
-@Command(name = "jbang", footer = "\nCopyright: 2020 Max Rydahl Andersen, License: MIT\nWebsite: https://github.com/jbangdev/jbang", description = "Compiles and runs .java/.jsh scripts.", subcommands = {
-		JbangRun.class, JbangEdit.class, JbangInit.class, JbangAlias.class, JbangTrust.class, JbangCache.class,
```

Comment:
To much bang here - lets just do `Run`, `Edit`, `Init`, `Alias`, `Trust` and `Cache`

they are already in jbang.cli. package.

Reply:
Sure, although you can never have enough bang for your buck :-D

Reply:
Done